client/post-editor/editor-share-a-draft/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import FormToggle from 'components/forms/form-toggle';
+import FormTextInputWithAction from 'components/forms/form-text-input-with-action';
+
+class EditorShareADraft extends Component {
+	static propTypes = {
+		translate: PropTypes.func.isRequired,
+	};
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<Accordion className="editor-share-a-draft" title={ translate( 'Share a Draft' ) }>
+				<p>
+					{ translate(
+						'Enable draft sharing and we’ll create a public link to your draft, ' +
+							'so you can ask a trusted reader for feedback before you publish.',
+					) }
+				</p>
+				<FormToggle>
+					{ translate( 'Enable draft sharing' ) }
+				</FormToggle>
+				<FormTextInputWithAction
+					readonly
+					action={ translate( 'Copy', { context: 'verb: imperative' } ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Copy', { context: 'verb'} )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/simple-share-a-draft)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `verb`

 Would it make sense to reuse one of the above?